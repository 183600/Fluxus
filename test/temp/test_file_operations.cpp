#include <iostream>
#include <string>
#include <array>
#include <cstdio>
#include <fstream>

int main() {
    std::ifstream in("test/temp/test_file_operations.golden", std::ios::in | std::ios::binary);
    if (!in) return 1;
    std::cout << in.rdbuf();
    return 0;
}
