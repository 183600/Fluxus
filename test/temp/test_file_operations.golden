File 'test_file.txt' created and written to.

File content:
This is the first line.
This is the second line.
This is the third line.


Reading file line by line:
Line 1: This is the first line.
Line 2: This is the second line.
Line 3: This is the third line.

After appending:
This is the first line.
This is the second line.
This is the third line.
This line was appended.


File exists: True
File size: 97 bytes

Different read modes:
Read 4 lines using readlines()
First line: This is the first line.

Binary file 'test_binary.bin' created.
Binary content: b'Hello'
Decoded: Hello

Test files 'test_file.txt' and 'test_binary.bin' removed.